src/query/interpreter.cpp

```diff
@@ -1183,53 +1243,79 @@ PreparedQuery PrepareIndexQuery(ParsedQuery parsed_query, bool in_explicit_trans
 
   auto label = interpreter_context->db->NameToLabel(index_query->label_.name);
   std::vector<storage::PropertyId> properties;
+  std::vector<std::string> properties_string;
   properties.reserve(index_query->properties_.size());
+  properties_string.reserve(index_query->properties_.size());
   for (const auto &prop : index_query->properties_) {
     properties.push_back(interpreter_context->db->NameToProperty(prop.name));
+    properties_string.push_back(prop.name);
   }
 
   if (properties.size() > 1) {
     throw utils::NotYetImplemented("index on multiple properties");
   }
 
+  Notification index_notification(SeverityLevel::INFO);
   switch (index_query->action_) {
     case IndexQuery::Action::CREATE: {
+      index_notification.code = NotificationCode::CREATE_INDEX;
+      index_notification.title = fmt::format("Created index on label {} on properties {}.", index_query->label_.name,
+                                             utils::Join(properties_string, ", "));
+
       handler = [interpreter_context, label, properties = std::move(properties),
-                 invalidate_plan_cache = std::move(invalidate_plan_cache)] {
+                 invalidate_plan_cache = std::move(invalidate_plan_cache)](Notification &index_notification) {
         if (properties.empty()) {
-          interpreter_context->db->CreateIndex(label);
+          if (!interpreter_context->db->CreateIndex(label)) {
+            index_notification.code = NotificationCode::EXISTANT_INDEX;
+            index_notification.title = "Index already exists.";
+          }
```

Comment:
Why do you use the properties only in the success case? If it provides useful information for the user, then use in both cases. If it doesn't provide useful information, then don't use it at all.

In my opinion that's not that important, but I don't know how the clients handle the notifications.

Reply:
I think you are right, will update the notifications message